examples/src/app.module.css

```diff
@@ -0,0 +1,78 @@
+html {
+  font-family: 'Roboto', sans-serif;
+}
+
+body {
+  margin: 0;
+}
+
+h3 {
+  margin-bottom: 0;
+  text-align: center;
+}
+
+h5 {
+  margin-top: 0;
+}
+
+.testApp {
+  display: flex;
+  flex-direction: column;
+  height: 100vh;
+}
+
+.example {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  height: 100%;
+}
+
+@media screen and (min-device-width: 1200px) {
```

Comment:
This should be moved to its own file/component. What do you think about making this a `<PhoneChrome>` component? :)